frame/evm/src/lib.rs

```diff
@@ -239,57 +322,25 @@ decl_module! {
 
 		fn deposit_event() = default;
 
-		const ModuleId: ModuleId = T::ModuleId::get();
-
-		/// Deposit balance from currency/balances module into EVM.
-		#[weight = 0]
-		fn deposit_balance(origin, value: BalanceOf<T>) {
```

Comment:
The deposit to the EVM module is gone, but shouldn't we have a deposit to EVM account to counter balance the withdraw ? (Or is it what mutate_account_basic should be used for ?)

Reply:
We can just directly use Substrate's transfer function, because now every EVM address has a corresponding Substrate address. Deposit function can be written, but it's redundant.